test/connectors/Acc45ConnectorSpec.scala

```diff
@@ -69,9 +68,10 @@ class Acc45ConnectorSpec extends SpecBase {
 
       "request is not successful due to business error - 'Exceeded maximum threshold of transactions'" in new Setup {
 
-        when[Future[HttpResponse]](mockHttpClient.POST(any, any, any)(any, any, any, any))
-          .thenReturn(Future.successful(HttpResponse(CREATED, casResponseStr03)))
-
+        when(requestBuilder.withBody(any())(any(), any(), any())).thenReturn(requestBuilder)
+        when(requestBuilder.setHeader(any[(String, String)]())).thenReturn(requestBuilder)
+        when(mockHttpClient.post(any)(any)).thenReturn(requestBuilder)
+        when(requestBuilder.execute(any, any)).thenReturn(Future.successful(HttpResponse(CREATED, casResponseStr03)))
         running(app) {
```

Comment:
I think we should add a new line between the when block and the running(app) just for readability and separation of interests.